images/macos/helpers/Common.Helpers.psm1

```diff
@@ -83,3 +83,16 @@ function Invoke-RestMethodWithRetry {
     )
     Invoke-RestMethod $Url -MaximumRetryCount 10 -RetryIntervalSec 30
 }
+
+function Invoke-ExpressionWithValidation {
```

Comment:
Any suggestions on better naming for this helper function?

Reply:
Invoke-ValidateCommand?

Reply:
Done